src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs

```diff
@@ -629,12 +629,15 @@ public void CheckHistoryPrevious()
             Assert.Equal(V, GetTextFromCurrentLanguageBuffer);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/4121")]
-        public void CheckHistoryAfterResetButtonClick()
+        [Fact]
+        public void CheckHistoryPreviousAfterReset()
         {
-            Task.Run(() => Window.Operations.ResetAsync(initialize: true)).PumpingWait();
-            Task.Run(() => Window.Operations.HistoryPrevious()).PumpingWait();
-            Assert.Equal("#reset", GetTextFromCurrentLanguageBuffer);
+            const string V = "#reset";
```

Comment:
Would it be worthwhile to check that there is only a single history entry for "#reset"?  Also, should there be a test of the history after using the reset button?